e2e/06-payments-receive-flow.e2e.spec.ts

```diff
@@ -81,7 +81,7 @@ describe("Receive Payment Flow", async () => {
       await shareButton.waitForDisplayed({ timeout: 8000 })
       await shareButton.click()
       const invoiceSharedScreen = await $('//*[contains(@name,"lntbs")]')
-      const invoiceSharedScreenElement = await invoiceSharedScreen.waitForDisplayed({
+      await invoiceSharedScreen.waitForDisplayed({
```

Comment:
there is another missing await there: https://github.com/GaloyMoney/galoy-mobile/pull/917/files#diff-6172ef87d6da592396503b953060c2c4cc11286af02fcb450fd398005b40e054R16

Reply:
Fixed